From 339bf978309591fa61211691f97139e5932f06f4 Mon Sep 17 00:00:00 2001
From: Matthias Maennich <maennich@google.com>
Date: Mon, 24 Jun 2019 18:10:19 +0100
Subject: ANDROID: gki/cuttlefish defconfigs: use prebuilt
 build-tools

Rather than relying on the hosts build tools, use a set of prebuilt
build-tools. These are tools like awk, cat, cp, hostname, python, rm,
sed, etc.

Bug: 135922132
Change-Id: Ie6b0bd208c684c9f153eb2f0bfac4712e02e8e05
Signed-off-by: Matthias Maennich <maennich@google.com>
---
 build.config.cuttlefish.aarch64 | 1 +
 build.config.cuttlefish.x86_64  | 1 +
 build.config.gki.aarch64        | 1 +
 build.config.gki.x86_64         | 1 +
 4 files changed, 4 insertions(+)

diff --git a/build.config.cuttlefish.aarch64 b/build.config.cuttlefish.aarch64
index bec1288d65e13..40c8d7a5219cc 100644
--- a/build.config.cuttlefish.aarch64
+++ b/build.config.cuttlefish.aarch64
@@ -8,6 +8,7 @@ EXTRA_CMDS=''
 KERNEL_DIR=common
 POST_DEFCONFIG_CMDS="check_defconfig"
 CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r353983c/bin
+BUILDTOOLS_PREBUILT_BIN=prebuilts/build-tools/path/linux-x86
 LINUX_GCC_CROSS_COMPILE_PREBUILTS_BIN=prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin
 FILES="
 arch/arm64/boot/Image.gz
diff --git a/build.config.cuttlefish.x86_64 b/build.config.cuttlefish.x86_64
index 33063abed0edb..c9496b0d2ce6a 100644
--- a/build.config.cuttlefish.x86_64
+++ b/build.config.cuttlefish.x86_64
@@ -8,6 +8,7 @@ EXTRA_CMDS=''
 KERNEL_DIR=common
 POST_DEFCONFIG_CMDS="check_defconfig"
 CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r353983c/bin
+BUILDTOOLS_PREBUILT_BIN=prebuilts/build-tools/path/linux-x86
 LINUX_GCC_CROSS_COMPILE_PREBUILTS_BIN=prebuilts/gcc/linux-x86/x86/x86_64-linux-android-4.9/bin
 FILES="
 arch/x86/boot/bzImage
diff --git a/build.config.gki.aarch64 b/build.config.gki.aarch64
index 408a2d1877e15..fce42ab0daa34 100644
--- a/build.config.gki.aarch64
+++ b/build.config.gki.aarch64
@@ -8,6 +8,7 @@ EXTRA_CMDS=''
 KERNEL_DIR=common
 POST_DEFCONFIG_CMDS="check_defconfig"
 CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r353983c/bin
+BUILDTOOLS_PREBUILT_BIN=prebuilts/build-tools/path/linux-x86
 LINUX_GCC_CROSS_COMPILE_PREBUILTS_BIN=prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin
 FILES="
 arch/arm64/boot/Image.gz
diff --git a/build.config.gki.x86_64 b/build.config.gki.x86_64
index 69d339dce4a0f..3ce45e962732b 100644
--- a/build.config.gki.x86_64
+++ b/build.config.gki.x86_64
@@ -8,6 +8,7 @@ EXTRA_CMDS=''
 KERNEL_DIR=common
 POST_DEFCONFIG_CMDS="check_defconfig"
 CLANG_PREBUILT_BIN=prebuilts-master/clang/host/linux-x86/clang-r353983c/bin
+BUILDTOOLS_PREBUILT_BIN=prebuilts/build-tools/path/linux-x86
 LINUX_GCC_CROSS_COMPILE_PREBUILTS_BIN=prebuilts/gcc/linux-x86/x86/x86_64-linux-android-4.9/bin
 FILES="
 arch/x86/boot/bzImage
-- 
2.23.0.444.g18eeb5a265-goog

